### PR TITLE
feat: add navigation menu

### PR DESCRIPTION
**Pull request by szymonograbek**

### CODE REVIEW

File: src/components/Menu/Menu.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    setFullScreenMenuVisible(isVisible);
+  };
+
+  return (
+    <div>
+      <header
+        className="fixed inset-x-0 top-0 z-10 flex h-[54px] items-center justify-center px-4 text-slate-900 transition-colors md:h-[60px] md:px-8"
+        style={{
+          backgroundColor: !displayFullScreenMenu ? '#ffffff' : undefined,
+          color: !displayFullScreenMenu ? '#0f172a' : '#ffffff',
```
Comment (kodowicz) [inline src/components/Menu/Menu.tsx:60]: please, make these colors as variables

File: src/components/Menu/MenuLinks.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  path: string;
+  onClick?: () => void;
+};
+
+function MenuButton({ label, path, onClick }: MenuButtonProps) {
+  return (
+    <motion.div
+      className="mb-10 text-xl font-semibold antialiased last:mr-0 md:mb-0 md:mr-10 md:text-lg"
+      whileHover={
+        window.innerWidth < 768 ? { translateX: 2 } : { translateY: -2 }
```
Comment (kodowicz) [inline src/components/Menu/MenuLinks.tsx:53]: I would make the breakpoint a variable

Comment (szymonograbek) [reply src/components/Menu/MenuLinks.tsx:53]: Done

File: src/components/Menu/Menu.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
+import { useEffect, useState } from 'react';
+import { AnimatePresence, motion } from 'framer-motion';
+import { MenuLinks } from './MenuLinks';
+import { Fade as Hamburger } from 'hamburger-react';
+import cx from 'classnames';
+import color from 'tailwindcss/colors';
+import { useLocation } from 'react-router-dom';
+
+export const MOBILE_BREAKPOINT = 768;
```
[the remark this answers is not in the document]
Comment (szymonograbek) [reply src/components/Menu/Menu.tsx]: Sure, I will export it

**Review by kodowicz — APPROVED** [review]

**Pull request merged**